docs/source/index.rst

```diff
@@ -1,7 +1,7 @@
 Introduction
 ============
 
-**cleanlab automatically finds and fixes errors in your ML datasets.**
+`cleanlab <https://github.com/cleanlab/cleanlab>`_ **automatically finds and fixes errors in your ML datasets.**
```

Comment:
@cgnorthcutt should we say: `finds and filters errors` instead of `finds and fixes errors`, since relabeling is not the focus in this repo?

Reply:
I think it's okay since cleaning out errors is a valid way to fix errors. If you want to be more precise you can change to "finds errors in any dataset and provides cleaned data for training."

Reply:
sticking with `finds and fixes errors` sounds better to me since it's more concise and explicitly mentions ML.  I agree it's a valid interpretation that filtering errors == fixing them.